dali/base/dadfs.cpp

```diff
@@ -6121,6 +6128,13 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
                 throw MakeStringException(-1, "The value of @numsubfiles (%d) is not equal to the number of SubFile items (%d)!",numSubfiles, subFileCount);
         }
     }
+    virtual bool isRestrictedAccess() override
```

Comment:
minor: could be const

Reply:
Unfortunately, I had to remove const from this function as it uses queryAttributes

Reply:
right yes annoying.